.travis.yml

```diff
@@ -10,7 +10,6 @@ install:
 
 script:
   - PYTHONWARNINGS=once coverage run --omit=*.virtualenvs*,*virtualenv* src/manage.py test --settings=sous_chef.settings_test member meal order notification delivery note billing page
-  - python3 src/manage.py test --settings=sous_chef.settings_test_fr member meal order notification delivery note billing page
```

Comment:
@lingxiaoyang sorry! I don't have pay attention to this "french" setting, it seems to be a double testing for me when I was looking into Travis, I will add the line again!